### PR TITLE
Add RWD to section--products

### PR DESCRIPTION
**Pull request by lkulbacki**
JIRA: https://projects.kodilla.com/browse/WDP181201-14, tam też dodałem ekstra komentarz.

### CODE REVIEW

**Comment by pawel-schmidt** [comment]
@lkulbacki please resolve merge conflicts.

File: src/index.html [hunk 1]
```diff
@@ -197,7 +197,7 @@ <h3>New furniture</h3>
       </div>
     </div>
     <div class="row">
-      <div class="col-3">
+        <div class="col-12 col-sm-6 col-md-4 col-lg-3">
```
Comment (pawel-schmidt) [inline src/index.html]: Line indented too much. The following you have changed - also.

File: src/sass/_bootstrap_custom.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* based on https://stackoverflow.com/a/48976550/3466464 */
```
Comment (pawel-schmidt) [inline src/sass/_bootstrap_custom.scss]: I guess we don't need this anymore. Am I right?

Comment (lkulbacki) [reply src/sass/_bootstrap_custom.scss]: Oh, and the current solution uses only predefined bootstrap breakpoints. Those defined in the custom file are exactly the same as original ones, with addition of "xs" breakpoint.

Comment (pawel-schmidt) [reply src/sass/_bootstrap_custom.scss]: IMHO if something is optional - it is redundant. It is easy to add some code "for future use", but in dynamically developed projects, this "future" sometimes never comes ;-) Thinking this way we are sure, the repo contains only necessary things. I really recommend to keeping files as slim as it is possible.

**Pull request merged**